api-tests/src/test/java/org/openmrs/module/reporting/dataset/definition/evaluator/RepeatPerTimePeriodDataSetEvaluatorTest.java

```diff
@@ -98,7 +98,7 @@ public void testEvaluate() throws Exception {
 
         // verify we delegated as expected
 
-        verify(service).evaluate(argThat(new ArgumentMatcher<DataSetDefinition>() {
+        verify(service).evaluate((DataSetDefinition) argThat(new ArgumentMatcher() {
```

Comment:
Somewhere in the upgrades to Mockito, the signature for `matches()` for `ArgumentMatcher<T>` was changed from `matches(Object argument)` to `matches(T argument)`. Using a non-generic `ArgumentMatcher` and casting the result allows us to use the same code with both old and new versions of Mockito.